packages/coil-anonymous-tokens/src/index.ts

```diff
@@ -249,13 +250,13 @@ export class AnonymousTokens implements AnonymousTokensService {
     }
   }
 
-  private async _populateTokensNow(coilAuthToken: string): Promise<void> {
+  private async _populateTokensNow(coilAuthToken?: string): Promise<void> {
```

Comment:
The changes in `@coil/client` don't seem to support working with either cookies or a token, why is this seemingly different?

Reply:
There's a bunch of errors as @coil/client is used outside the extension.

On that note, if you wanted to, you could just make token optional for the moment in all the client helper methods, until we decide to update all calling sites (at some later point - adding a cookie impl to the client ). In terms of calling sites, there's a command line script that connects via user/pass and streams to a payment pointer specified on the command line for example. Probably a few others.